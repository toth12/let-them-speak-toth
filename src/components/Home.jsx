import React from 'react';
import { Link } from 'react-router-dom';

const Home = props => (
  <div className='home background-image'>
    <div className='top'>
      <div className='brand' />
      <div>
        <div className='welcome'>A data-driven anthology of oral history interviews with Holocaust survivors</div>
      </div>
      <Link className='button brown' to='/about'>Let's explore</Link>
    </div>
    <div className='bottom'>
      <hr/>
      <p>A project by Gabor Mihaly Toth in collaboration with the Yale Digital Humanities Lab</p>
    </div>
  </div>
)

export default Home;