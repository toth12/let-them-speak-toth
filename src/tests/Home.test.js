import React from 'react';
import Home from '../components/Home';
import { createBrowserHistory } from 'history'
import { Router } from 'react-router';
import renderer from 'react-test-renderer';

it('renders', () => {
  const tree = renderer
    .create(
      <Router history={createBrowserHistory()}>
        <Home/>
      </Router>
    ).toJSON();
  expect(tree).toMatchSnapshot();
});