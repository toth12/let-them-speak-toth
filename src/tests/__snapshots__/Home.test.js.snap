// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="home background-image"
>
  <div
    className="top"
  >
    <div
      className="brand"
    />
    <div>
      <div
        className="welcome"
      >
        A data-driven anthology of oral history interviews with Holocaust survivors
      </div>
    </div>
    <a
      className="button brown"
      href="/about"
      onClick={[Function]}
    >
      Let's explore
    </a>
  </div>
  <div
    className="bottom"
  >
    <hr />
    <p>
      A project by Gabor Mihaly Toth in collaboration with the Yale Digital Humanities Lab
    </p>
  </div>
</div>
`;
